src/main/scala/com/johnsnowlabs/nlp/embeddings/BertEmbeddings.scala

```diff
@@ -402,46 +401,41 @@ trait ReadBertTensorflowModel extends ReadTensorflowModel {
 
   addReader(readTensorflow)
 
-  def loadSavedModel(tfModelPath: String, spark: SparkSession): BertEmbeddings = {
+  def loadSavedModel(modelPath: String, spark: SparkSession): BertEmbeddings = {
 
-    val f = new File(tfModelPath)
-    val savedModel = new File(tfModelPath, "saved_model.pb")
+    val detectedEngine = modelSanityCheck(modelPath)
```

Comment:
We will have to merge this with the remote loading of the model:

https://github.com/JohnSnowLabs/spark-nlp/pull/12931/files#diff-13282f334c1c1e8d9a1253a31cce7d87da16659719b1b606f251e6688e420a75R39

Reply:
@DevinTDHa We will. I am refactoring all of the loadSavedModel first, then I will see how your DBFS/S3 implementation.

I suggest you don't refactor, make one of them work/compatible with DBFS/S3/HDFS, and once you are done testing I can see the best place to add those changes